lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -558,6 +558,21 @@ export class BrowserCacheManager extends CacheManager {
         return "";
     }
 
+    clearMsalCookies(): void {
+        const cookiePrefix = `${Constants.CACHE_PREFIX}.${this.clientId}`;
+        const cookieList = document.cookie.split(";");
+        for (let i = 0; i < cookieList.length; i++) {
```

Comment:
Can we do `cookieList.forEach(cookie => {})` to avoid the Object Injection Sink warning below?

Reply:
Done.